cmd/goal/clerk.go

```diff
@@ -145,7 +146,13 @@ var sendCmd = &cobra.Command{
 		if txFilename == "" {
 			// Sign and broadcast the tx
 			wh, pw := ensureWalletHandleMaybePassword(dataDir, walletName, true)
-			tx, err := client.SendPaymentFromWallet(wh, pw, fromAddressResolved, toAddressResolved, fee, amount, noteBytes, closeToAddressResolved)
+			var tx transactions.Transaction
+
+			if firstValid != 0 && lastValid != 0 {
+				tx, err = client.SendPaymentFromWalletWithCustomValidity(wh, pw, fromAddressResolved, toAddressResolved, fee, amount, noteBytes, closeToAddressResolved, basics.Round(firstValid), basics.Round(lastValid))
+			} else {
+				tx, err = client.SendPaymentFromWallet(wh, pw, fromAddressResolved, toAddressResolved, fee, amount, noteBytes, closeToAddressResolved)
+			}
```

Comment:
I think we should complain if either `firstValid` or `lastValid` are set _and not both_.

Reply:
Implementing it as a print-error-and-exit